submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/config/GeneralConfig.kt

```diff
@@ -86,46 +87,47 @@ class GeneralConfig {
 
     @Bean
     @ConditionalOnProperty(prefix = "app.cluster", name = ["enabled"], havingValue = "true")
-    fun clusterClient(properties: ApplicationProperties): ClusterClient {
-        return when (properties.cluster.default) {
+    fun clusterClient(properties: ApplicationProperties): ClusterClient =
+        when (properties.cluster.default) {
             LSF -> lsfCluster(properties)
             SLURM -> slurmCluster(properties)
         }
-    }
 
     @Bean
     @ConditionalOnProperty(prefix = "app.cluster", name = ["enabled"], havingValue = "false")
     fun localClusterClient(): ClusterClient = LocalClusterClient()
 
     @Bean
-    fun ftpClient(properties: ApplicationProperties) = ftpClient(properties.security.filesProperties)
+    fun userFilesFtpClient(properties: ApplicationProperties) = ftpClient(properties.security.filesProperties.ftp)
```

Comment:
I think we should use @Qualifier to avoid depending of bean function name

Reply:
Good suggestion, I didn't know we could use constants. I used the opportunity to improve a couple more of qualified beans we had lying around